cdk/getApiUrl.js

```diff
@@ -0,0 +1,4 @@
+// Prints the Lambda Function URL of the Club Site API Lambda
+// This is used as an environment variable when building the React client
+const cdk_out = require("./cdk.out.json");
+console.log(cdk_out["HackClubSite-ServiceStack"].ClubSiteApiFunctionUrl);
```

Comment:
Why do we need a separate file just to print the lambda function? Is this for sanity check purposes?

Reply:
To keep things simple, we are using a [Lambda function URL](https://docs.aws.amazon.com/lambda/latest/dg/lambda-urls.html) instead of API Gateway. The function URL should not change once created, but to keep things consistent, we want to specify this URL as an environment variable for the client build ([Events.js:9](https://github.com/HackAtUCI/Club-Site/blob/ac4cf6fdda3ac0796ee1bde8c97c173bc508dcc1/client/src/app/views/events/Events.js#L9)). The deployment workflow uses the `getApiUrl` script to set the function URL as a step output ([deploy-main.yml:42-43](https://github.com/HackAtUCI/Club-Site/blob/ac4cf6fdda3ac0796ee1bde8c97c173bc508dcc1/.github/workflows/deploy-main.yml#L42-L43)) which is later used during the client build step ([deploy-main.yml:54](https://github.com/HackAtUCI/Club-Site/blob/ac4cf6fdda3ac0796ee1bde8c97c173bc508dcc1/.github/workflows/deploy-main.yml#L54)).